### ISSUE TITLE
Support for CBS350-48T-4G and|or CBS250-48T-4G

### ISSUE THREAD

**Issue by pantaras** [napalm-automation-community/napalm-s350#75]
### Short description
Hello, 
Does anybody has the information whether these 2 models are/will be supported CBS350-48T-4G,  CBS250-48T-4G

I'm primary interested in "napalm_install_config" function or similar alternative. 
My current use case is for cisco2960 switches is: 
```
 tasks:
 - name: Push Config Changes
     napalm_install_config: 
            dev_os: ios
```
Really appreciate the answer

**Comment by qaxi** [comment]
Hello,
to add at least basic support we need text output of those commands, to add new device:
```
show arp
show startup-config
show version
show system
show inventory
show hosts
show interface status
show running-config
show interfaces status
show interfaces description
show ports jumbo-frame
show ip int
show lldp neighbors
show sntp status
```

Those commands may need parameter change to suit your configuration:
```
show running-config full
show lldp local FastEthernet1
show lldp neighbors FastEthernet0/1
```

Please add each command to separate file - command `show arp` to file named `show_arp.txt`

Make pull request to /napalm-s350/tree/develop/test/unit/mocked_data/`your device type`/
or if you prefer make a Issue and add files as an atachement.

Support for `napalm_install_config` is on the way.

**Comment by qaxi** [comment]
solved by #80
